src/openrct2/core/DataSerialiserTraits.h

```diff
@@ -703,19 +703,20 @@ template<> struct DataSerializerTraitsT<TrackDesignMazeElement>
 {
     static void encode(OpenRCT2::IStream* stream, const TrackDesignMazeElement& val)
     {
-        uint32_t temp = ByteSwapBE(val.all);
-        stream->Write(&temp);
+        stream->Write(&val.location);
+        stream->Write(&val.mazeEntry);
     }
     static void decode(OpenRCT2::IStream* stream, TrackDesignMazeElement& val)
     {
-        uint32_t temp;
-        stream->Read(&temp);
-        val.all = ByteSwapBE(temp);
```

Comment:
Does removing the swap break backwards compatibility?

From what I can see, `ByteSwapBE` swaps every byte, so for example the x and y coords would be swapped around whereas passing the new location field doesn't do this.

Reply:
This only concerns network serialisation. It does require a network version bump, which I’ll do before merging, but otherwise it does not break backwards compatibility.